lib/ansible/module_utils/compat/selinux.py

```diff
@@ -0,0 +1,103 @@
+# Copyright: (c) 2021, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import os
+import sys
+
+from ansible.module_utils.common.text.converters import to_native, to_bytes
+from ctypes import CDLL, c_char_p, c_int, byref, POINTER, get_errno
+
+try:
+    _selinux_lib = CDLL('libselinux.so.1', use_errno=True)
```

Comment:
I'm curious what's the `.1`, will it ever increase over time?

Reply:
Dunno- it hasn't in the 8 years since the original code was written... ;)